demisto_sdk/__main__.py

```diff
@@ -311,7 +311,7 @@ def validate(config, **kwargs):
 @click.option('-a', '--artifacts_path', help='Destination directory to create the artifacts.',
               type=click.Path(file_okay=False, resolve_path=True))
 @click.option('--zip/--no-zip', help='Zip content artifacts folders', default=True)
-@click.option('--content-packs', help='Create only content_packs artifacts.', default=False)
+@click.option('--packs', help='Create only content_packs artifacts.', is_flag=True)
```

Comment:
why are you changing the argument name?

Reply:
This was the default argument name in the previous version.